src/DependencyGraph.java

```diff
@@ -0,0 +1,169 @@
+import java.util.*;
```

Comment:
Needs to be in a package otherwise the module system will yell at you. Doesn't matter what it is `dev.mccue.resolve.whatever`